src/main/java/net/cyberkitsune/prefixchat/LocalizedString.java

```diff
@@ -19,7 +21,7 @@ public static FileConfiguration getLocaleFile()
             KitsuneChat.getInstance().mcLog.severe("[KitsuneChat] Localization resource not found!!!");
             return null;
         }
-        InputStreamReader reader = new InputStreamReader(inputStream);
+        InputStreamReader reader = new InputStreamReader(inputStream, StandardCharsets.UTF_8);
```

Comment:
Unrelated fix. OK for now but good practice to make sure pull requests only address one issue